src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -209,6 +211,32 @@ def subvolume_getpath(self, **kwargs):
             ret = self.volume_exception_to_retval(ve)
         return ret
 
+    def subvolume_info(self, **kwargs):
+        ret        = None
+        volname    = kwargs['vol_name']
+        subvolname = kwargs['sub_name']
+        groupname  = kwargs['group_name']
+
+        try:
+            with open_volume(self, volname) as fs_handle:
+                with open_group(fs_handle, self.volspec, groupname) as group:
+                    with open_subvol(fs_handle, self.volspec, group, subvolname) as subvolume:
+                        mon_addr_lst = []
+                        r, mon_map_json, err = self.mgr.mon_command({'prefix': 'mon dump', 'format': 'json'})
+                        if r != 0:
+                            return -1, "", "Couldn't fetch mon addrs, mon dump cmd failed"
```

Comment:
extremely minor nit: use an error (probably `ENODATA`) rather than -1?

(it's ok if this be fixed in a follow on PR)

Reply:
Done